Sources/SwiftFormat/Utilities/FileIterator.swift

```diff
@@ -135,7 +135,7 @@ public struct FileIterator: Sequence, IteratorProtocol {
         else {
           break
         }
-        path = destination
+        path = URL(fileURLWithPath: destination, isDirectory: false, relativeTo: item).path
```

Comment:
The symlink could point to a directory, right? I would just remove the `isDirectory` bit so that URL can infer that information from the file system.

Reply:
I just fixed in 5f90fb3d8646b74946bd9917c72d49d8acdeba16, but I don't think this change is meaningful because [the existing code](https://github.com/swiftlang/swift-format/blob/06ca47b4769b9b59028d71b6c3b0ab4360edb35f/Sources/SwiftFormat/Utilities/FileIterator.swift#L152) always treats it as `isDirectory: false`.